system_metrics_collector/test/system_metrics_collector/test_linux_memory_measurement.cpp

```diff
@@ -89,12 +91,8 @@ constexpr const std::array<const char *, 10> kSamples = {
 class TestLinuxMemoryMeasurementNode : public system_metrics_collector::LinuxMemoryMeasurementNode
 {
 public:
-  TestLinuxMemoryMeasurementNode(
-    const std::string & name,
-    const std::chrono::milliseconds measurement_period,
-    const std::string & publishing_topic,
-    const std::chrono::milliseconds publish_period)
-  : LinuxMemoryMeasurementNode(name, measurement_period, publishing_topic, publish_period),
+  TestLinuxMemoryMeasurementNode(const std::string & name, const rclcpp::NodeOptions & options)
+  : LinuxMemoryMeasurementNode(name, options),
```

Comment:
```suggestion
  : LinuxMemoryMeasurementNode{name, options},
```

Reply:
Doesn't look like the braces were applied here

Reply:
Now it's really done :)